README.md

```diff
@@ -47,8 +47,8 @@ LPVS will start scanning automatically, then provide comments about the licenses
 
 1. Install SCANOSS Python package by following the [guideline](https://github.com/scanoss/scanoss.py#installation).
 
-2. Fill in `licenses.json` file with the information about permitted, restricted, and prohibited licenses (mandatory) as well as their compatibility specifics (optional). 
-A template of the `licenses.json` file can be found in the repository at `src/main/resources/licenses.json`.
+2. Create necessary MySQL database and user and fill in `licenses` and `license_conflicts` tables with the information about permitted, restricted, and prohibited licenses (mandatory) as well as their compatibility specifics (optional). 
```

Comment:
It would be great to automatically setup the database table and necessary information if `lpvs` database is not configured in MySQL.